reana_client/api/client.py

```diff
@@ -93,17 +93,16 @@ def run_analysis(self, user, organization, reana_spec):
                                   organization=organization,
                                   reana_spec=json.loads(json.dumps(
                                       reana_spec, sort_keys=True))).result()
-
-            if http_response.status_code == 200:
+            if http_response.status_code == 201:
                 return response
             else:
                 raise Exception(
                     "Expected status code 200 but replied with "
```

Comment:
Should the Exception message be `Expected status code 201 but <REANA_SERVER> replied with ...`?